prow/config.yaml

```diff
@@ -52,19 +65,18 @@ presubmits:
     rerun_command: "/test pull-kops-e2e-kubernetes-aws"
     trigger: "(?m)^/test( all| pull-kops-e2e-kubernetes-aws),?(\\s+|$)"
   kubernetes/kubernetes:
-  - name: pull-kubernetes-bazel-build
-    agent: kubernetes
-    context: pull-kubernetes-bazel-build
-    always_run: true
-    rerun_command: "/test pull-kubernetes-bazel-build"
-    trigger: "(?m)^/test( all| pull-kubernetes-bazel-build),?(\\s+|$)"
-    skip_branches:
-    - release-1.4
-    - release-1.5
-    - release-1.6  # doesn't have BUILD files under //vendor/k8s.io
+  - <<: &pull-kubernetes-bazel-build-job
+      name: pull-kubernetes-bazel-build
+      agent: kubernetes
+      context: pull-kubernetes-bazel-build
+      always_run: true
+      rerun_command: "/test pull-kubernetes-bazel-build"
+      trigger: "(?m)^/test( all| pull-kubernetes-bazel-build),?(\\s+|$)"
+    skip_branches: *all-release-branches
```

Comment:
```
branches: 
- master
```
is easier? so we don't need the all-release-branches anymore

Reply:
The counterargument is that kubernetes is starting to experiment with feature branches.

A part of me wishes that either the skipped or targeted branches could be a regex, but then that introduces other problems.

Reply:
That's a pretty simple change if we think it's a good idea.

Reply:
I think making those regexes might not be a bad idea. It would certainly be useful here. I think defining `all-release-branches` once is pretty fine though.